lib/beaker/options/parser.rb

```diff
@@ -205,7 +205,8 @@ def update_option(key, value, source)
       #   3.  the 'CONFIG' section of the hosts file
       #   4.  options file values
       #   5.  subcommand options, if executing beaker subcommands
-      #   6.  default or preset values are given the lowest priority
+      #   6.  options from $HOME/.beaker/subcommands_options.yaml
```

Comment:
@kevpl yes the same was pointed out by @ferglor in his review.
I'm changing this to be a part of `SubcommandOptionsParser` rather than `OptionsFileParser`